src/PrestaShopBundle/Resources/views/Admin/Module/Includes/modal_read_more_content.html.twig

```diff
@@ -118,6 +118,25 @@
                 {{ 'No description found for this module :('|trans({}, 'Admin.Modules.Notification') }}
               {% endif %}
             </p>
+            {% if module.attributes.multistoreCompatibility is not null %}
+              <div class="module-readmore-multistore-content">
+                <h3>{{ 'Multistore compatibility:'|trans({}, 'Admin.Modules.Notification') }}</h3>
+                {% if module.attributes.multistoreCompatibility === constant('\\Module::MULTISTORE_COMPATIBILITY_YES') %}
+                  {{ 'Yes: this module is compatible with the multistore feature. It can be either:'|trans({}, 'Admin.Modules.Notification') }}
+                  <ul>
+                    <li>{{ 'configured differently from one store to another;'|trans({}, 'Admin.Modules.Notification') }}</li>
+                    <li>{{ 'configured quickly in the same way on all stores thanks to the all shops context or to the group of shops;'|trans({}, 'Admin.Modules.Notification') }}</li>
+                    <li>{{ 'or even activated for one store and deactivated for another.'|trans({}, 'Admin.Modules.Notification') }}</li>
+                  </ul>
+                {% elseif module.attributes.multistoreCompatibility === constant('\\Module::MULTISTORE_COMPATIBILITY_PARTIAL') %}
+                  {{ 'Partially: this module is partially compatible with the multistore feature. Some of its options might not be available.'|trans({}, 'Admin.Modules.Notification') }}
+                {% elseif module.attributes.multistoreCompatibility === constant('\\Module::MULTISTORE_COMPATIBILITY_NOT_CONCERNED') %}
+                  {{ 'Not concerned: this module is not compatible with the multistore feature because it wouldn\'t be useful.'|trans({}, 'Admin.Modules.Notification') }}
+                {% elseif module.attributes.multistoreCompatibility === constant('\\Module::MULTISTORE_COMPATIBILITY_NO') %}
+                  {{ 'No: this module is not compatible with the multistore feature. It means that its configuration applies for all stores.'|trans({}, 'Admin.Modules.Notification') }}
+                {% endif %}
```

Comment:
Where is the else condition ?

Reply:
true there is none, I should use several if

Reply:
So if `module.attributes.multistoreCompatibility` is not in these cases (as you don't use a setter/getter for checking that), you can have an empty page.

Reply:
not really, it just doesn't display the multistore compatibility, which is fine, I want to display the multistore compatibility information only if the information is in the module.

But I did implement your getter / setter suggestion 👍